build/utils.go

```diff
@@ -26,8 +25,15 @@ const (
 	mobyHostGatewayName = "host-gateway"
 )
 
+// isURL returns true if the provided str is an HTTP(S) URL by checking if it
+// has a http:// or https:// scheme. No validation is performed to verify if the
+// URL is well-formed.
+func isURL(str string) bool {
```

Comment:
`isHTTPURL` or similar would be a better name here. We also have `isRemoteURL` so would be good to avoid confusion for this.

Reply:
Ah, yeah, I could rename; I went for keeping it as close to the original, but its a horrible function in either case, and even considered just inlining the code to even more discourage using it.

Reply:
Updated; PTAL 🤗